db/migrate/20160701180402_add_sortable_version_to_puppet_modules.rb

```diff
@@ -13,6 +13,8 @@ def sortable_version(version)
   end
 
   def up
+    return unless Object.const_defined?('::Katello::PuppetModule')
+
     add_column :katello_puppet_modules, :sortable_version, :string
```

Comment:
can you move this return below the add_column?

Reply:
the reason i ask, is that its better to leave the structural changes to the db as is, but any data handling is fine to skip